Test to check duplicate registries with same name but different capitalisation (#1247)
wazuh/wazuh-qa: docs/tests/integration/test_fim/test_registry/test_registry_basic_usage/test_basic_usage_registry_duplicated_entries.md

# Test duplicated regitry entries

> This test will check that for two monitored registries with the same name value but with different capitalisation only triggers one added event when the registry is created.

Reviewer: Typo in word capitalization. 
```suggestion
This test will check that for two monitored registries with the same name value but with different capitalization only triggers one added event when the registry is created.
```

Author: Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1247/commits/1c9bfd087282e2435cc7f5f770ca5786abe95752)

> This test will check that for two monitored registries with the same name value but with different capitalisation only triggers  one modified event when the registry is changed.

Reviewer 2: ```suggestion
This test will check that for two monitored registries with the same name value but with different capitalisation only triggers one modified event when the registry is changed.
```

## General info

> |Tier | Number of tests | Time spent| Test file |
> |:--:|:--:|:--:|:--:|
> | 0 | 1 | 0:0:25 | [test_basic_usage_registry_duplicated_entries.py](../../../../../../tests/integration/test_fim/test_registry/test_registry_basic_usage/test_basic_usage_registry_duplicated_entries.py)|

Reviewer 2: ```suggestion
|Tier | Number of tests | Time spent|
|:--:|:--:|:--:|
| 0 | 1 | 25s |
```

> # Test duplicated registry entries
> 
> This test will check that two monitored registries with the same name value but with different capitalization only triggers one added event when the registry is created.
> 
> ## General info
> 
> |Tier | Number of tests | Time spent|
> |:--:|:--:|:--:|
> | 0 | 1 | 15s |
> 
> ## Test logic
> 
> The test monitor two registries with the same path but different capitalization. It creates
> one registry with the path being one of the two registries monitorized and then tries to grab
> one added event for the registry creation. Finally it tries to grab the added event one second
> time but it should rise one TimeoutError to ensure only one added event was sent.

Reviewer 3: ```suggestion
# Test duplicated registry entries

This test will check that two monitored registries with the same name value but with different capitalization only triggers one added event when the registry is created.

## General info

|Tier | Number of tests | Time spent|
|:--:|:--:|:--:|
| 0 | 1 | 15s |

## Test logic

The test monitor two registries with the same path but different capitalization. It creates
one registry with the path being one of the two registries monitored and then tries to grab
one added event for the registry creation. Finally, it tries to grab the added event one second
time but it should raise one TimeoutError to ensure only one added event was sent.
```

Author: Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1247/commits/c89eb367365e53d034a3f04058598377b3acc2c6)

> ## Execution result

Reviewer 2: ```suggestion
## Execution result

```

# Test duplicated regitry entries

This test will check that for two monitored registries with the same name value but with different capitalisation only triggers one modified event when the registry is changed.

## General info

|Tier | Number of tests | Time spent|
|:--:|:--:|:--:|
| 0 | 1 | 25s |

## Test logic

The test creates two registries with the same name but different capitalization,
modifies one of them and waits for the monitor to grab that event.
The second call to the monitor should arise TimeoutError to be successful. This test

> ensures the windows agent doesn't duplicate alerts due to windows being case insensitive.

Reviewer 4: ```suggestion
ensures the Windows agent doesn't duplicate alerts due to Windows being case insensitive.
```

Author: Documentation changed in [commit](https://github.com/wazuh/wazuh-qa/pull/1247/commits/2ae38fa0530f59f33378bda5978f2bbcb45d015c)
